runtime/jcl/common/system.c

```diff
@@ -490,20 +490,20 @@ jobject getPropertyList(JNIEnv *env)
 
 #if defined(OPENJ9_BUILD) && JAVA_SPEC_VERSION == 8
 	/* Set the maximum direct byte buffer allocation property if it has not been set manually */
-	if ((UDATA) -1 == javaVM->directByteBufferMemoryMax) {
+	if (~(UDATA)0 == javaVM->directByteBufferMemoryMax) {
```

Comment:
Perhaps this is picking nits, but we should have parentheses here to remove the assumption that `~` has higher precedence than `==` (here and elsewhere):
```c
	if ((~(UDATA)0) == javaVM->directByteBufferMemoryMax) {
```